src/ripple/app/tx/impl/Escrow.cpp

```diff
@@ -147,7 +147,10 @@ EscrowCreate::preflight(PreflightContext const& ctx)
             return temMALFORMED;
         }
 
-        if (condition->type != Type::preimageSha256)
+        // Conditions other than PrefixSha256 require the
```

Comment:
We could consider not reverting this part of the commit.  Then if someone enables CryptoConditionsSuite it will have no effect on the code.  But I don't feel strongly about it.